AK: Split Sponsorship by Chamber

braykuka opened the pull request:
- issue [1232](https://github.com/openstates/issues/issues/1232)
- split sponsorship by chamber
- fix the conference_committee_list_url issue

braykuka commented:
@NewAgeAirbender Please review it.

braykuka commented:
@jessemortenson Please review it.

jessemortenson commented:
Thanks, code looks OK, but I am seeing some sponsorship data that is expected but not showing up in the JSON output data. In the Open States database, we have at least one sponsor for almost all the bills in the Alaska (AK) session `33`. (only one bill in the DB is missing a sponsor).

In the JSON data produced by the scrape on this branch, I am seeing 275 files where `sponsorships` is an empty array. So for 274 of those bills/files, I am expecting at least one sponsor to be in that array.

Here's the data that's in the database currently for those bills, to give you an idea of what I'm expecting:

```
+----------+----------------------------------------------+------------+-------+
|identifier|sponsor_name                                  |entity_type |primary|
+----------+----------------------------------------------+------------+-------+
|HB 105    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SJR 21    |SENATE TRANSPORTATION                         |organization|true   |
|SB 193    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 265    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 399    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 315    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HR 6      |HOUSE RULES                                   |organization|true   |
|HCR 26    |HOUSE RULES                                   |organization|true   |
|HB 190    |HOUSE WAYS & MEANS                            |organization|true   |
|HB 313    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 289    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SCR 34    |SENATE FINANCE                                |organization|true   |
|SCR 40    |SENATE RULES                                  |organization|true   |
|SCR 21    |SENATE RESOURCES                              |organization|true   |
|HB 181    |HOUSE JUDICIARY                               |organization|true   |
|HB 132    |HOUSE JUDICIARY                               |organization|true   |
|HB 131    |HOUSE JUDICIARY                               |organization|true   |
|HR 4      |HOUSE RULES                                   |organization|true   |
|HB 393    |HOUSE RESOURCES                               |organization|true   |
|SB 76     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 71     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 40     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 117    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 50     |SENATE RESOURCES                              |organization|true   |
|SB 224    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 226    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SCR 23    |SENATE FINANCE                                |organization|true   |
|HB 59     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 58     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SCR 101   |SENATE RULES                                  |organization|true   |
|SB 120    |SENATE EDUCATION                              |organization|true   |
|SB 145    |SENATE LABOR & COMMERCE                       |organization|true   |
|HB 199    |HOUSE TRANSPORTATION                          |organization|true   |
|SB 65     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SSCR 10   |SENATE RULES                                  |organization|true   |
|SB 86     |SENATE FINANCE                                |organization|true   |
|HB 79     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 87     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 83     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 105    |SENATE TRANSPORTATION                         |organization|true   |
|HB 92     |HOUSE FISHERIES                               |organization|true   |
|HB 85     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SCR 29    |SENATE LABOR & COMMERCE                       |organization|true   |
|SB 97     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 85     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 58     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 202    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HCR 20    |HOUSE RULES                                   |organization|true   |
|HB 130    |HOUSE JUDICIARY                               |organization|true   |
|HB 140    |HOUSE RULES BY REQUEST                        |organization|true   |
|HB 119    |HOUSE RULES BY REQUEST                        |organization|true   |
|SB 123    |SENATE TRANSPORTATION                         |organization|true   |
|SB 66     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 155    |SENATE FINANCE                                |organization|true   |
|SSCR 8    |SENATE RULES                                  |organization|true   |
|SB 126    |SENATE LABOR & COMMERCE                       |organization|true   |
|HB 158    |HOUSE MILITARY & VETERANS' AFFAIRS            |organization|true   |
|SB 107    |SENATE FINANCE                                |organization|true   |
|SB 41     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 122    |SENATE RULES                                  |organization|true   |
|HB 54     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SCR 18    |SENATE COMMUNITY & REGIONAL AFFAIRS           |organization|true   |
|SCR 28    |SENATE JUDICIARY                              |organization|true   |
|HB 49     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 165    |HOUSE WAYS & MEANS                            |organization|true   |
|HB 41     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 75     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 114    |SENATE RULES                                  |organization|true   |
|SB 64     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 68     |SENATE RESOURCES                              |organization|true   |
|HB 178    |HOUSE FINANCE                                 |organization|true   |
|SB 98     |SENATE FINANCE                                |organization|true   |
|SB 262    |SENATE STATE AFFAIRS                          |organization|true   |
|HB 74     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 81     |SENATE RULES BY REQUEST OF LEGISLATIVE COUNCIL|organization|true   |
|SJR 8     |SENATE RESOURCES                              |organization|true   |
|HB 141    |HOUSE RULES                                   |organization|true   |
|SCR 26    |SENATE HEALTH & SOCIAL SERVICES               |organization|true   |
|SSCR 11   |SENATE RULES                                  |organization|true   |
|SB 234    |SENATE LABOR & COMMERCE                       |organization|true   |
|HB 128    |HOUSE TRANSPORTATION                          |organization|true   |
|SB 183    |SENATE LABOR & COMMERCE                       |organization|true   |
|HB 312    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 54     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 160    |HOUSE WAYS & MEANS                            |organization|true   |
|HJR 8     |HOUSE WAYS & MEANS                            |organization|true   |
|HCR 22    |HOUSE RULES                                   |organization|true   |
|HCR 18    |HOUSE LABOR & COMMERCE                        |organization|true   |
|SB 96     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SCR 31    |SENATE HEALTH & SOCIAL SERVICES               |organization|true   |
|HB 397    |HOUSE STATE AFFAIRS                           |organization|true   |
|SB 49     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SSCR 3    |SENATE RULES                                  |organization|true   |
|HR 7      |HOUSE TRANSPORTATION                          |organization|true   |
|HB 193    |HOUSE FINANCE                                 |organization|true   |
|HB 288    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 228    |SENATE LABOR & COMMERCE                       |organization|true   |
|SSCR 2    |SENATE RULES                                  |organization|true   |
|HB 294    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 210    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HR 2      |HOUSE RULES                                   |organization|true   |
|HCR 4     |HOUSE FINANCE                                 |organization|true   |
|SB 80     |SENATE RULES BY REQUEST OF LEGISLATIVE COUNCIL|organization|true   |
|SB 188    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SCR 13    |SENATE JUDICIARY                              |organization|true   |
|SB 93     |SENATE LABOR & COMMERCE BY REQUEST            |organization|true   |
|SCR 12    |SENATE HEALTH & SOCIAL SERVICES               |organization|true   |
|HB 66     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 200    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 199    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 50     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 269    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HCR 25    |HOUSE FINANCE                                 |organization|true   |
|SB 52     |SENATE EDUCATION                              |organization|true   |
|HCR 19    |HOUSE STATE AFFAIRS                           |organization|true   |
|HB 171    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HCR 21    |HOUSE LABOR & COMMERCE                        |organization|true   |
|HB 76     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 148    |HOUSE EDUCATION                               |organization|true   |
|HB 217    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SCR 2     |SENATE LABOR & COMMERCE                       |organization|true   |
|SJR 7     |SENATE RESOURCES                              |organization|true   |
|SCR 4     |SENATE HEALTH & SOCIAL SERVICES               |organization|true   |
|HB 400    |HOUSE EDUCATION                               |organization|true   |
|SCR 1     |SENATE RULES                                  |organization|true   |
|HB 129    |HOUSE JUDICIARY                               |organization|true   |
|SCR 6     |SENATE FINANCE                                |organization|true   |
|HB 117    |HOUSE RULES                                   |organization|true   |
|SB 156    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 39     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SJR 19    |SENATE RESOURCES                              |organization|true   |
|SB 42     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 258    |SENATE STATE AFFAIRS                          |organization|true   |
|SB 237    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 57     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 216    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HR 3      |HOUSE RULES                                   |organization|true   |
|HR 1      |HOUSE RULES                                   |organization|true   |
|HB 42     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 186    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 48     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 59     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HR 5      |HOUSE RULES                                   |organization|true   |
|HB 106    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 86     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 40     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 133    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 208    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 184    |SENATE STATE AFFAIRS BY REQUEST               |organization|true   |
|SSCR 7    |SENATE RULES                                  |organization|true   |
|SB 243    |SENATE RESOURCES                              |organization|true   |
|SB 240    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 190    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 146    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 198    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 69     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 381    |HOUSE WAYS & MEANS                            |organization|true   |
|HB 376    |HOUSE TRANSPORTATION                          |organization|true   |
|HB 344    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 314    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 368    |HOUSE ENERGY                                  |organization|true   |
|HB 333    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 343    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 186    |HOUSE LABOR & COMMERCE                        |organization|true   |
|SB 192    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 159    |HOUSE LABOR & COMMERCE                        |organization|true   |
|HCR 15    |HOUSE HEALTH & SOCIAL SERVICES                |organization|true   |
|HB 396    |HOUSE RESOURCES                               |organization|true   |
|SB 249    |SENATE LABOR & COMMERCE                       |organization|true   |
|SB 203    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 290    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 276    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 281    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 386    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SCR 27    |SENATE HEALTH & SOCIAL SERVICES               |organization|true   |
|HCR 24    |HOUSE RULES                                   |organization|true   |
|HB 275    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 182    |SENATE LABOR & COMMERCE                       |organization|true   |
|SB 266    |SENATE EDUCATION                              |organization|true   |
|SB 209    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 297    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 212    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 295    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HCR 23    |HOUSE RULES                                   |organization|true   |
|HCR 27    |HOUSE RULES                                   |organization|true   |
|SCR 30    |SENATE RESOURCES                              |organization|true   |
|HB 307    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 204    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 394    |HOUSE RESOURCES                               |organization|true   |
|HB 146    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HJR 19    |HOUSE TRANSPORTATION                          |organization|true   |
|SB 223    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HCR 17    |HOUSE RULES                                   |organization|true   |
|HB 154    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SCR 22    |SENATE LABOR & COMMERCE                       |organization|true   |
|SSCR 6    |SENATE RULES                                  |organization|true   |
|SSCR 1    |SENATE RULES                                  |organization|true   |
|SCR 24    |SENATE FINANCE                                |organization|true   |
|SSCR 5    |SENATE RULES                                  |organization|true   |
|SCR 44    |SENATE FINANCE                                |organization|true   |
|SCR 36    |SENATE RULES                                  |organization|true   |
|SCR 25    |SENATE FINANCE                                |organization|true   |
|SSCR 4    |SENATE RULES                                  |organization|true   |
|SJR 20    |SENATE COMMUNITY & REGIONAL AFFAIRS           |organization|true   |
|SCR 20    |SENATE LABOR & COMMERCE                       |organization|true   |
|SCR 17    |SENATE RESOURCES                              |organization|true   |
|SSCR 9    |SENATE RULES                                  |organization|true   |
|SB 260    |SENATE FINANCE                                |organization|true   |
|SB 196    |SENATE LABOR & COMMERCE                       |organization|true   |
|HB 293    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 70     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 191    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 278    |HOUSE WAYS & MEANS                            |organization|true   |
|HB 67     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 143    |HOUSE RESOURCES                               |organization|true   |
|HB 388    |HOUSE RESOURCES                               |organization|true   |
|SB 217    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 256    |SENATE STATE AFFAIRS BY REQUEST               |organization|true   |
|HB 306    |HOUSE STATE AFFAIRS BY REQUEST                |organization|true   |
|HJR 7     |HOUSE WAYS & MEANS                            |organization|true   |
|SCR 10    |SENATE FINANCE                                |organization|true   |
|HB 299    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 389    |HOUSE TRANSPORTATION                          |organization|true   |
|HB 271    |HOUSE LABOR & COMMERCE                        |organization|true   |
|SJR 18    |SENATE FINANCE                                |organization|true   |
|SB 241    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 211    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 387    |HOUSE RESOURCES                               |organization|true   |
|SB 264    |SENATE COMMUNITY & REGIONAL AFFAIRS           |organization|true   |
|HB 268    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 274    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HJR 29    |HOUSE TRANSPORTATION                          |organization|true   |
|HB 390    |HOUSE RESOURCES                               |organization|true   |
|HB 296    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 398    |HOUSE RULES                                   |organization|true   |
|HB 375    |HOUSE TRANSPORTATION                          |organization|true   |
|HB 277    |HOUSE WAYS & MEANS                            |organization|true   |
|HB 189    |HOUSE LABOR & COMMERCE                        |organization|true   |
|SB 138    |SENATE STATE AFFAIRS                          |organization|true   |
|HB 60     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 255    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 84     |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HJR 27    |HOUSE LABOR & COMMERCE                        |organization|true   |
|SB 194    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 185    |SENATE STATE AFFAIRS                          |organization|true   |
|HB 392    |HOUSE RESOURCES                               |organization|true   |
|SB 225    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 259    |SENATE RULES BY REQUEST OF LEGISLATIVE COUNCIL|organization|true   |
|HB 270    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 68     |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SCR 14    |SENATE RESOURCES                              |organization|true   |
|SB 263    |SENATE TRANSPORTATION                         |organization|true   |
|SCR 19    |SENATE FINANCE                                |organization|true   |
|SCR 33    |SENATE STATE AFFAIRS                          |organization|true   |
|SCR 15    |SENATE JUDICIARY                              |organization|true   |
|SCR 32    |SENATE FINANCE                                |organization|true   |
|SB 261    |SENATE STATE AFFAIRS                          |organization|true   |
|SB 153    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|SB 125    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HJR 28    |HOUSE JUDICIARY                               |organization|true   |
|HCR 16    |HOUSE RULES                                   |organization|true   |
|HB 125    |HOUSE RESOURCES                               |organization|true   |
|SSCR 12   |SENATE RULES                                  |organization|true   |
|SB 204    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
|HB 273    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 257    |SENATE RESOURCES                              |organization|true   |
|HB 395    |HOUSE TRANSPORTATION                          |organization|true   |
|SB 113    |SENATE FINANCE                                |organization|true   |
|HB 282    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|HB 391    |HOUSE EDUCATION                               |organization|true   |
|SCR 11    |SENATE FINANCE                                |organization|true   |
|SCR 16    |SENATE LABOR & COMMERCE                       |organization|true   |
|HB 272    |HOUSE RULES BY REQUEST OF THE GOVERNOR        |organization|true   |
|SB 187    |SENATE RULES BY REQUEST OF THE GOVERNOR       |organization|true   |
+----------+----------------------------------------------+------------+-------+

I'm attaching the scraper output JSON directory in case you need that for reference.
[240926_ak_scrape.zip](https://github.com/user-attachments/files/17159123/240926_ak_scrape.zip)

Further, looking at the JSON scrape data, there are zero sponsorships where `entity_type` is `organization` (only `person`), so I'm guessing something wrong with the code that is excluding organizational sponsors.

---

A smaller issue: Some of the sponsor names include the text `BY REQUEST`. Please change to remove that, so `KIEHL BY REQUEST` becomes `KIEHL`.

braykuka commented:
@jessemortenson There was an issue to get the sponsors. Please review it now.

jessemortenson commented on scrapers/ak/bills.py:
```diff
@@ -120,21 +140,20 @@ def scrape_bill(
                     spons_str = spons_str + " Committee (by request of the governor)"
-
                 if spons_str:
                     bill.add_sponsorship(
```
Can you please add logic here that adds the `chamber` argument for bills sponsored by an organization? It looks like every organization name starts with either `SENATE` or `HOUSE`, so if `SENATE` then `chamber=upper` and if `HOUSE` then `chamber=lower`

jessemortenson commented on scrapers/ak/bills.py:
```diff
@@ -120,21 +140,20 @@ def scrape_bill(
                 # Committee sponsorship
                 spons_str = spons_str.strip()
-
                 if re.match(r" BY REQUEST OF THE GOVERNOR$", spons_str):
```
Looks like about half of the organizational sponsorships still include the `by request...` language in the `name` property. Can you please check this and adjust logic to try to strip it out in all cases?

```
jesse@greenbookwork:~/repo/openstates/braykuka/openstates-scrapers/scrapers/_data/ak$ rg -ic "by request" | wc -l
122
```

```
 SELECT bill_identifier, "name", "primary", organization_id  FROM bill_sponsorships WHERE entity_type = 'organization' AND lower(name) LIKE '%request%' ORDER BY bill_identifier, name;
┌─────────────────┬────────────────────────────────────────────────┬─────────┬───────────────────────────────────────────────────────────────────┐
│ bill_identifier │                      name                      │ primary │                          organization_id                          │
├─────────────────┼────────────────────────────────────────────────┼─────────┼───────────────────────────────────────────────────────────────────┤
│ HB105           │ HOUSE RULES BY REQUEST OF THE GOVERNOR         │ true    │ "~{\"name\": \"HOUSE RULES BY REQUEST OF THE GOVERNOR\"}"         │
│ HB106           │ HOUSE RULES BY REQUEST OF THE GOVERNOR         │ true    │ "~{\"name\": \"HOUSE RULES BY REQUEST OF THE GOVERNOR\"}"         │
│ HB119           │ HOUSE RULES BY REQUEST                         │ true    │ "~{\"name\": \"HOUSE RULES BY REQUEST\"}"                         │
│ HB140           │ HOUSE RULES BY REQUEST                         │ true    │ "~{\"name\": \"HOUSE RULES BY REQUEST\"}"                         │
│ HB146           │ HOUSE RULES BY REQUEST OF THE GOVERNOR         │ true    │ "~{\"name\": \"HOUSE RULES BY REQUEST OF THE GOVERNOR\"}"         │
```

so:

- `HOUSE RULES BY REQUEST OF THE GOVERNOR` should just be `HOUSE RULES`
- `HOUSE RULES BY REQUEST` should just be `HOUSE RULES`
- `SENATE RULES BY REQUEST OF LEGISLATIVE COUNCIL` should just be `SENATE RULES`

jessemortenson reviewed (changes requested):
Thank you for fixing the missing organizational sponsorships. I have two more small requests regarding how those org sponsorships are coming through.

jessemortenson commented on scrapers/ak/bills.py:
```diff
@@ -93,20 +98,35 @@ def scrape_bill(
                         entity_type="person",
+                        chamber=sponsor_chamber,
                         classification="primary",
                         primary=True,
```
Oops, one more issue I found is that sometimes the scraper says that two sponsors are primary, even though only one sponsor should be primary in those cases. What I noticed is that this happens when there are both House and Senate sponsors for a bill (upper and lower): in that case the first person in each list is marked `primary` by the scraper.

Instead, it should be: if a House bill, only first House sponsor should be primary=True, and if a Senate bill, only the first Senate sponsor should be primary=True

EG 

  - only WRIGHT should be a primary sponsor on https://www.akleg.gov/basis/Bill/Detail/33?Root=HB126
  - only KIEHL should be a primary sponsor on https://www.akleg.gov/basis/Bill/Detail/33?Root=sb10

braykuka commented:
@jessemortenson Thanks for your review. I've fixed those issues now. 
Actually I couldn't see the sponsor's chamber value in the json files though I've added it in the code. 
![image](https://github.com/user-attachments/assets/81299355-aba8-4365-8cd1-a164c72fd545)
Please let me know if you have still issues.

jessemortenson reviewed (approved):
Looks good to me, thank you!

jessemortenson commented:
> @jessemortenson Thanks for your review. I've fixed those issues now. Actually I couldn't see the sponsor's chamber value in the json files though I've added it in the code. ![image](https://private-user-images.githubusercontent.com/81768784/372554938-81299355-aba8-4365-8cd1-a164c72fd545.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgwNzQ3MTgsIm5iZiI6MTcyODA3NDQxOCwicGF0aCI6Ii84MTc2ODc4NC8zNzI1NTQ5MzgtODEyOTkzNTUtYWJhOC00MzY1LThjZDEtYTE2NGM3MmZkNTQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA0VDIwNDAxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBhMDhiM2EwNDRiNzc4YzAwYTA1ZWJmOWE4MjQzZGM1ZTc1ZGY1NDE1NDg5NGFlMzJlMzYxNTlkOTcwODJjMjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.UyWh6gMOQC4xIfiI6LmUEJPr9VdnnAk8RxE0b5CEPGI) Please let me know if you have still issues.

Ah, I think you may need to sync updates from the upstream repository (and run `poetry install` again). We have added more recent changes to the `openstates-core` library that causes these chamber values to be emitted in the JSON files, and only more recent versions of `openstates-scrapers` have specified the newest version of `openstates-core`.  (I ran these test scrapes with `openstates=6.20.5` in order to get chamber values showing up in the JSON output). We have been iterating actively on that code in recent weeks.
